files/ko/web/javascript/guide/expressions_and_operators/index.html

```diff
@@ -478,449 +746,557 @@ <h3 id="논리_연산자">논리 연산자</h3>
+<h3 id="comma_operator">쉼표 연산자</h3>
 
-<p><a href="/ko/docs/Web/JavaScript/Reference/Operators/Comma_Operator">쉼표 연산자</a> (<code>,</code>)는 두 피연산자를 평가하고, 마지막 피연산자의 값을 반환합니다. 이 연산자는 주로 <code>for</code> 반복문 안에서 각각의 시간에 복수의 변수들을 갱신하기 위하여 사용합니다.</p>
+<p>
+  <a href="/ko/docs/Web/JavaScript/Reference/Operators/Comma_Operator">쉼표 연산자</a>(<code>,</code>)는 두 피연산자를
+  모두 평가하고, 오른쪽 피연산자의 값을 반환합니다. 쉼표 연산자는 주로 <code>for</code> 반복문 안에서 사용해서 한 번의
+  반복으로 다수의 변수를 변경할 때 사용합니다. 그 외의 상황에 사용하는 것은, 꼭 필요하지 않다면 좋지 않은 코드 스타일로
+  여겨집니다. 대개 쉼표 연산자보다는 두 개의 분리된 명령문을 사용하는 편이 낫습니다.
+</p>
 
-<p>예를 들어, a는 각 변에 10개의 원소가 있는 2차원 배열일때, 다음의 코드는 콤마 연산자를 두 변수를 한번에 증가 시키기 위하여 사용하였습니다. 이 코드는 배열의 대각선에 위치한 원소를 출력합니다:</p>
+<p>
+  쉼표 연산자의 예제로는, <code>a</code>가 2차원 배열로서 10행 10열의 요소를 가지고 있다면, 다음 코드는 쉼표 연산자를
+  사용해서 두 개의 변수를 한꺼번에 업데이트하는 모습을 보입니다. 코드의 실행 결과는 <code>a</code>의 요소를 대각선((0,
+  0), (1, 1), (2, 2), ...)으로 출력한 것입니다.
+</p>
 
-<pre><code>var x = [0,1,2,3,4,5,6,7,8,9]
-var a = [x, x, x, x, x];
+<pre class="brush: js">
+var x = [0,1,2,3,4,5,6,7,8,9]
+var a = [x, x, x, x, x]
 
 for (var i = 0, j = 9; i &lt;= j; i++, j--)
-  console.log('a[' + i + '][' + j + ']= ' + a[i][j]);</code></pre>
+//                                   ^
+  console.log('a[' + i + '][' + j + ']= ' + a[i][j])
+</pre>
 
-<h3 id="단항_연산자">단항 연산자</h3>
+<h3 id="unary_operators">단항 연산자</h3>
 
-<p>단항 연산자는 오직 하나의 피연산자를 가지고 연산을합니다.</p>
+<p>단항 연산은 오직 하나의 피연산자만 사용하는 연산입니다.</p>
 
-<h4 id="delete" name="delete"><code>delete</code></h4>
+<h4 id="delete"><code>delete</code></h4>
 
-<p><code><a href="/ko/docs/Web/JavaScript/Reference/Operators/delete">delete</a></code>연산자는 객체, 객체의 속성 또는 배열의 특정한 위치에 있는 객체를 삭제합니다. 문법은 다음과 같습니다:</p>
+<p>
+  <code><a href="/ko/docs/Web/JavaScript/Reference/Operators/delete">delete</a></code
+  >연산자는 객체의 속성을 삭제합니다. 구문은 다음과 같습니다.
+</p>
 
-<pre class="brush: js">delete objectName;
-delete objectName.property;
+<pre class="brush: js">
+delete object.property;
+delete object[propertyKey];
 delete objectName[index];
-delete property; // legal only within a with statement
+delete property; // with문 안에서만 유효함
 </pre>
 
-<p><code>objectName</code>은 객체의 이름이고, <code>property</code>는 객체에 존재하는 속성이고,<code>index</code>는 배열의 위치를 나타내는 정수입니다.</p>
-
-<p>네번째 형태의 경우,객체의 속성을 삭제하기 위하여 <code><a href="/ko/docs/Web/JavaScript/Reference/Statements/with">with</a></code> 구문이 있어야만 동작합니다.</p>
-
-<p><code>delete</code> 연산자를 <code>var</code> 구문을 사용하지 않고 묵시적으로 만들어진 변수를 삭제할 때 사용할 수 있습니다.</p>
-
-<p>만약 <code>delete</code>연산자의 동작이 성공했다면, delete 연산자는 속성이나, 원소를 <code>undefined</code> 로 설정합니다. <code>delete</code> 연산자는 연산이 수행 가능할때 true값을 반환합니다; 수행이 불가능 할 경우 <code>false</code> 값을 반환합니다.</p>
-
-<pre class="brush: js">x = 42;
+<p>
+  위의 구문에서, <code>objectName</code>은 객체의 이름이고, <code>property</code>는 객체에 존재하는 속성,
+  <code>propertyKey</code>는 존재하는 속성을 가리키는 문자열이나 심볼입니다.
+</p>
+
+<p>
+  네 번째 형태의 경우에는, 전역 객체의 속성을 삭제하는 것이 아니라면
+  <code><a href="/ko/docs/Web/JavaScript/Reference/Statements/with">with</a></code>
+  명령문이 있어야만 유효합니다.
```

Comment:
```suggestion
  네 번째 형태의 경우에는, 객체 그리고 전역 객체의 속성을 삭제하기 위해서라면
  <code><a href="/ko/docs/Web/JavaScript/Reference/Statements/with">with</a></code>
  명령문 안에 있어야만 유효합니다.
```
오역이 된 듯합니다. 확인 부탁드릴게요 !  

원문은 아래와 같습니다.
```
<p>The fourth form is legal only within a
  <code><a href="/en-US/docs/Web/JavaScript/Reference/Statements/with">with</a></code>
  statement, to delete a property from an object, and also for properties of the global
  object.</p>
```

Reply:
지금 보니 원문 문장이... 좀 요상하긴 한데요,

> is legal only within a `with` statement,

다음의 경우에는 `with`문 안에 있어야 유효합니다.

> to delete a property from an object,

객체의 속성을 삭제할 때는요.

> and also for properties of the global object.

*(그리고)* 전역 객체의 속성을 삭제할 때도 유효합니다. *(`with` 없이도)*

콘솔에서 간단하게 테스트 하실 수 있어요.
```js
window.a = 'foo'
delete a
window.a // undefined
```

[엄격 모드에서는 `SyntaxError`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete#strict_vs._non-strict_mode)니까 아예 빠지는게 맞을 것 같긴 한데,,

Reply:
mdn/content#6896

본 PR이 먼저 병합되면 다음에 맞추고, 6896이 먼저 병합되면 커밋 하나 더 올릴게요.

Reply:
넵 기다리고 있겠습니다 : )